index.html

```diff
@@ -7,33 +7,241 @@
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img4.png" alt="This is an image of a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img5.png" alt="This is an image of a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img6.png" alt="This is an image of a laptop and a phone." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
     </div>
   </section>
   
+  <!--About Myself container-->
+  <section class="section-container">
+    <div class="section-sub">
+      <h1 class="title dark-blue">About</h1>
```

Comment:
- [ ] To follow `HTML` best practices, it's recommended to use only one `h1` tag per page. Therefore, it's best to avoid using multiple `h1` tags and instead use other heading tags, such as `h2, h3`, and so on, to structure the content on your page. **This not only makes your code more semantically correct but also helps improve accessibility and** [SEO](https://developers.google.com/search/docs/fundamentals/seo-starter-guide#:~:text=SEO%20%2D%20Search%20engine%20optimization%3A%20the,our%20presence%20on%20the%20web.). 🔍

  _Please, apply this suggestion for all of your multiple `h1` tags on your entire page._